php/class-plugin.php

```diff
@@ -36,7 +36,6 @@ public function __construct() {
 	 */
 	public function init() {
 		$this->config = apply_filters( 'customize_pane_resizer_plugin_config', $this->config, $this );
-		$this->min = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
```

Comment:
The tests are passing, so I'm not sure which symlinks you're referring to? They have all been replaced as of Sunday.